02-Custom Login/Auth0Sample/LoginViewController.m

```diff
@@ -0,0 +1,159 @@
+//
+//  LoginViewController.m
+// Auth0Sample
+//
+// Copyright (c) 2016 Auth0 (http://auth0.com)
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#import <Foundation/Foundation.h>
+#import "LoginViewController.h"
+#import "ProfileViewController.h"
+#import "Auth0-Swift.h"
+#import "Auth0InfoHelper.h"
+#import "NSString_EmailValidation.h"
+#import "UIViewController_Dismiss.h"
+#import "SignUpViewController.h"
+#import "UIStoryboardSegueWithCompletion.h"
+
+@interface LoginViewController()
+
+@property (nonatomic, weak) IBOutlet UITextField *loginEmailText;
+@property (nonatomic, weak) IBOutlet UITextField *loginPasswordText;
+@property (nonatomic, weak) IBOutlet UIActivityIndicatorView *spinner;
+@property (nonatomic, weak) IBOutlet UIButton* loginButton;
+
+@end
+
+@implementation LoginViewController
+
+- (void) getUserFromCredentials:(A0Credentials*) credentials callback:(void (^ _Nonnull)(NSError * _Nullable, A0UserProfile * _Nullable))callback
```

Comment:
Let's avoid the word `get` here. We could use `loadUserWithCredentials:callback:` instead. Also `with` is a better fit than `from` in this case, in my opinion.